tests/test_load_rep.py

```diff
@@ -42,7 +42,7 @@ def test_load_rep_data(self):
         with self.store.session_scope():
             # there must be states after the import
             states = self.store.session.query(self.store.db_classes.State).all()
-            self.assertEqual(len(states), 746)
+            self.assertEqual(len(states), 16)
```

Comment:
this is a very large drop in the number of State measurements.  Are rejecting allow those rows?

Reply:
When Enhanced Validator is used, only `rep_test1.rep` and `sen_ssk_freq.dsf` files are imported. And the latter doesn't have any State objects. That's why there is a very large drop and most of the rows(files) are rejected.

Reply:
I can't see the failures for `sen_tracks.rep` or `uk_track.rep`. 

Aah, it's a bug:
```
[
    {
        "REP Importer - Enhanced Validation Error": "Difference between Bearing (359.251) and Heading (22.500) is more than 90 degrees!"
    }
]
```

Those angles are only 23 degrees apart. We can merge this, I'll create an issue to cover the bug.